src/core/Breadcrumbs/Breadcrumbs.test.tsx

```diff
@@ -79,14 +86,7 @@ it('should accept currentIndex prop', () => {
 });
 
 it('should overflow when there is not enough space', () => {
-  const scrollWidthSpy = jest
-    .spyOn(HTMLElement.prototype, 'scrollWidth', 'get')
-    .mockReturnValueOnce(250)
-    .mockReturnValue(200);
-  const offsetWidthSpy = jest
-    .spyOn(HTMLElement.prototype, 'offsetWidth', 'get')
-    .mockReturnValue(200);
-
+  jest.spyOn(UseOverflow, 'useOverflow').mockReturnValue([jest.fn(), 2]);
```

Comment:
You are already spying above. Cant you save that spy in variable and just change the return value?